backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1021,25 +1021,14 @@ databaseChangeLog:
                     nullable: false
                     foreignKeyName: fk__organization__ordering_provider
                     referencedTableName: provider
-        - addColumn:
```

Comment:
This is wrong, so I'm correcting my past mistake here. This column was actually renamed in the changes, so the correct move is not to create the new column and drop the old one, but rather to just rename the column back. (The copy/paste was wrong anyway, because the FK references `api_audit_event`, which doesn't exist yet at this point)